### PR TITLE
docs(adapters): Add prisma client docs for typescript users

### LINKED ISSUE #824: Property `prisma` doesn't exist on `Global & typeof globalThis`
**Your question**
When setting up the PrismaClient I used the following code in my `[...nextauth].ts` file

```typescript
export let prisma: PrismaClient

if (process.env.NODE_ENV === 'production') {
  prisma = new PrismaClient()
} else {
  if (!global.prisma) { <--
    global.prisma = new PrismaClient() <--
  }
  prisma = global.prisma <--
}
```

I keep getting this error/warning tho.
`Property 'prisma' does not exist on type 'Global & typeof globalThis'.ts(2339)`

Trying to understand how to get rid of this error. It seems to work anyways since the client doesn't keep opening new connections, but I'm a bit annoyed with the error. 

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [x] Did not find the example project helpful

### PR DESCRIPTION
**Pull request by vedantnn71**
## ☕️ Reasoning
Added documentation/code snippet for connecting prisma client w/ prisma adapter for typescript users as the code mentioned in the documentation doesn't work out of the box with typescript.

## 🧢 Checklist

- [x] Documentation
- [x] Tests
- [x] Ready to be merged

## 🎫 Affected issues

Fixes: #824

## 📌 Resources

- [Contributing guidelines](./CONTRIBUTING.md)
- [Code of conduct](./CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

**Review by balazsorban44 — CHANGES_REQUESTED** [review]
let's just convert this:

https://github.com/nextauthjs/next-auth/blob/a4ada77ce80c6ec32a6dda3ec255e6baf42abe51/docs/docs/adapters/prisma.md?plain=1#L17-L24

to TypeScript

**Comment by vedantnn71** [comment]
Converted ✅

File: docs/docs/adapters/prisma.md [hunk 1]
```diff
@@ -14,13 +14,17 @@ npm install prisma --save-dev
 
 Create a file with your Prisma Client:
 
-```javascript title="lib/prismadb.js"
-import { PrismaClient } from "@prisma/client"
+```typescript title="lib/prismadb.ts"
+import { PrismaClient } from "@prisma/client";
```
Comment (balazsorban44) [inline docs/docs/adapters/prisma.md:18]:
```suggestion
import { PrismaClient } from "@prisma/client"
```

File: docs/docs/adapters/prisma.md [hunk 2]
```diff
@@ -14,13 +14,17 @@ npm install prisma --save-dev
...
+
+declare global {
+  var prisma: PrismaClient | undefined
+}
 
 const client = globalThis.prisma || new PrismaClient()
-if (process.env.NODE_ENV !== "production") globalThis.prisma = client
+if (process.env.NODE_ENV !== "production") globalThis.prisma = client;
```
Comment (balazsorban44) [inline docs/docs/adapters/prisma.md:25]:
```suggestion
if (process.env.NODE_ENV !== "production") globalThis.prisma = client
```

File: docs/docs/adapters/prisma.md [hunk 3]
```diff
@@ -14,13 +14,17 @@ npm install prisma --save-dev
...
 
-export default client
+export default client;
```
Comment (balazsorban44) [inline docs/docs/adapters/prisma.md:27]:
```suggestion
export default client
```

**Review by balazsorban44 — APPROVED** [review]

**Pull request merged**